src/points.rkt

```diff
@@ -201,7 +201,7 @@
       (let ([curr (map (compose <-bf (call f)) pts)]
             [good? (map (call pre) pts)])
         (if (and prev (andmap (λ (good? old new) (or (not good?) (=-or-nan? old new repr))) good? prev curr))
-            (map (λ (good? x) (if good? x +nan.0)) good? curr)
+            (map (λ (good? x) (if good? x (real->repr +nan.0 repr))) good? curr)
```

Comment:
I got confused the first time I read this. Is this correct? `+nan.0` is used as a sentinel output for "ignore this value", even if it's not a valid value of that representation. Yes, this is dumb.